synapse/tools/migrate_020.py

```diff
@@ -517,6 +517,10 @@ class Migrator(s_base.Base):
         if self.srcdedicated is None:
             self.srcdedicated = False
 
+        self.destdedicated = conf.get('destdedicated')
```

Comment:
`conf.get('destdedicated', False)`

Reply:
Argparse returns {arg: None} if the arg is not specified.  Although I changed these to store_true, so I could just .get it now.